telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/command/youtube/service/ChatIdForYoutubeService.java

```diff
@@ -0,0 +1,31 @@
+package com.github.bumblebee.command.youtube.service;
+
+import com.github.bumblebee.command.youtube.dao.ChatIdForYoutubeRepository;
+import com.github.bumblebee.command.youtube.entity.ChatId;
+import com.google.api.client.util.Lists;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+
+@Service
+@Transactional
+public class ChatIdForYoutubeService {
```

Comment:
Naming could be better. You can create YoutubeSubscriptionService and put this logic here.